explorer/src/lib/components/footer/Footer.css

```diff
@@ -0,0 +1,67 @@
+.footer {
+  display: flex;
+  flex-direction: row;
+  padding: 1.25rem 0;
+  font-size: 0.875rem;
+  justify-content: flex-end;
+}
+
+.footer__copyright {
+  order: 1;
+}
+
+.footer__links {
+  padding-left: 1.25rem;
+  padding-right: 1.875rem;
+  display: flex;
+  gap: 1.25rem;
+  order: 2;
+}
+
+.footer__links-link {
+  color: var(--primary-color);
+}
+
+.footer__links-link:visited {
+  color: var(--primary-color);
+}
+
+.footer__links-link:hover {
+  color: var(--secondary-color);
+}
+
+.footer__logo {
+  order: 3;
+}
+
+.footer__logo img {
+  width: 5.375rem;
+}
+
+@media (max-width: 768px) {
+  .footer {
+    flex-wrap: wrap;
+    justify-content: space-between;
+    gap: 0.625rem;
+  }
+
+  .footer__copyright {
+    order: 3;
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .footer__links {
+    padding: 0;
+    order: 2;
+  }
+
+  .footer__logo {
+    order: 1;
+  }
+
+  .footer__logo img {
```

Comment:
Ideally this would be `.footer__logo-image` and the `App` image be passed the class name since following the BEM convention, [all styles should be via class name selectors](https://en.bem.info/methodology/css/#combining-a-tag-and-a-class-in-a-selector).

Reply:
I do agree with this, there is a similar issue with the image in the statistics pane (but have approved the PR anyways, as anyways there will be work on the map and the code around it will change).

Reply:
I will be adding static tests to eslint for BEM and there is also a [VS Code plugin](https://marketplace.visualstudio.com/items?itemName=Box-Of-Hats.bemhelper&source=post_page-----43f12def6d33--------------------------------) with Svelte support currently in their development branch so we can catch other violations.